sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -599,7 +599,7 @@ public async Task VerifyAppService2019RequestMockAsync()
         [Test]
         public async Task AllAppServiceEnvVarsSetSelects2019Api()
         {
-            using var environment = new TestEnvVar(new() { { "MSI_ENDPOINT", "https://mock.msi.endpoint/" }, { "MSI_SECRET", "mock-msi-secret" }, { "IDENTITY_ENDPOINT", "https://identity.endpoint/" }, { "IDENTITY_HEADER", "mock-identity-header" }, { "AZURE_POD_IDENTITY_AUTHORITY_HOST", null } });
```

Comment:
This change is due to the fix in MSAL 4.67.1 which added support for Azure Machine Learning MSI. ML MSI emulates App Service 2017, which populates MSI_SECRET.

Reply:
Turns out that this was a regression. Waiting for 4.67.2 which should address this bug.